libheif/context.cc

```diff
@@ -2304,6 +2304,15 @@ Error HeifContext::encode_image(const std::shared_ptr<HeifPixelImage>& pixel_ima
     }
       break;
 
+    case heif_compression_HTJ2K: {
```

Comment:
Minor thing: maybe this `case` can be simply combined with the `heif_compression_JPEG2000` above.

Reply:
Will see if we need to add something extra for the HT case, and combine if not.

Reply:
Have combined it.